File: Shuttle.Sentinel.donejs/src/permissions.js
var Permissions = {
    Manage: {
        Roles: 'sentinel://roles/manage',
        Users: 'sentinel://users/manage',
        Messages: 'sentinel://messages/manage'
    },
    View: {
        Dashboard: 'sentinel://dashboard/view',
        Subscriptions: 'sentinel://subscriptions/view',
        Queues: 'sentinel://queues/view',
        DataStores: 'sentinel://data-stores/view',
        Roles: 'sentinel://roles/view',
        Users: 'sentinel://users/view'
    }
};

export default Permissions;

File: Shuttle.Sentinel/Permissions.cs
namespace Shuttle.Sentinel
{
    public class Permissions
    {
        public static class Manage
        {
            public const string Messages = "sentinel://messages/manage";
            public const string Monitoring = "sentinel://monitoring/manage";
            public const string Queues = "sentinel://queues/manage";
            public const string Schedules = "sentinel://schedules/manage";
            public const string Subscriptions = "sentinel://subscriptions/manage";
        }
        
        public static class View
        {
            public const string Monitoring = "sentinel://monitoring/view";
            public const string Queues = "sentinel://queues/view";
            public const string Schedules = "sentinel://schedules/view";
            public const string Subscriptions = "sentinel://subscriptions/view";
        }
    }
}